### PR TITLE
Prevented scanning cere blockchain

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: chains/substrate/listener.go [hunk 1]
```diff
@@ -76,10 +76,10 @@ func (l *listener) start() error {
 	}
 
 	go func() {
-		err := l.pollBlocks()
-		if err != nil {
-			l.log.Error("Polling blocks failed", "err", err)
-		}
+		// err := l.pollBlocks()
```
Comment (AndreiNavoichyk) [inline chains/substrate/listener.go]: We should add a comment on why this part has been commented on.

Comment (AndreiNavoichyk) [reply chains/substrate/listener.go]: We should try to disable scanning earlier. Please, research.

Comment (karthikeyan-cere) [reply chains/substrate/listener.go]: Added comment. 
I tried to disable it in the place of connecting to the substrate. But it throws registering a new router error. 
That's why I have disabled it at the place of scanning.

Comment (AndreiNavoichyk) [reply chains/substrate/listener.go]: > But it throws registering a new router error

Have you researched for what does it mean?

Comment (karthikeyan-cere) [reply chains/substrate/listener.go]: During initialization, the system will fetch whitelist id from blockchin. So we can't stop at that point. So I have commented on the block polling point.

File: config/config.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ type RawChainConfig struct {
 	Id       string            `json:"id"`       // ChainID
 	Endpoint string            `json:"endpoint"` // url for rpc endpoint
 	From     string            `json:"from"`     // address of key to use
+	Scan     bool              // should scan
```
Comment (AndreiNavoichyk) [inline config/config.go]: Follow the existing structure. Add json path.

Comment (karthikeyan-cere) [reply config/config.go]: Updated

**Pull request merged**